var/www/skitter.fanfic.dev/index.html

```diff
@@ -1,88 +1,111 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang=en>
 	<head>
 		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<title>skitter.fanfic.dev</title>
-		<style type="text/css">
-			body { max-width: 960px; }
-			body { font-size: 1.2em; }
-			body h2 { font-size: 1.5em; }
-			.u { font-weight: bold; }
-			.c { font-family: monospace; }
-		</style>
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Karla&family=Source+Code+Pro&display=swap" rel="stylesheet">
+		<link rel="stylesheet" href="style.css">
 	</head>
 	<body>
 		<main>
 			<h1>skitter.fanfic.dev</h1>
-			<p>
+			<h3>
```

Comment:
Dropping a comment here, but in general I'm not sure you're using the correct markup. Some specifics:
* is this really a header?
* you're using `<code></code>` for monospace but the actual json in this page is in non-monospace `<blockquote></blockquote>`
* this doesn't pass a validator, but mine isn't free of typos either lol. We should fix it.
* we should probably have a header before the general error section
* we might want headers above each endpoint as well

Reply:
- is this really a header?
  - I am assuming you're talking about the `<h1>` ? Yes
- you're using `<code></code>` for monospace but the actual json in this page is in non-monospace `<blockquote></blockquote>`
  - I think I talked about code and monospace in an earlier comment. As for the actual JSON, it's not semantic HTML but a random design choice.
- this doesn't pass a validator, but mine isn't free of typos either lol. We should fix it.
  - I fixed the typos I could see but statements like this `status 200: Body is simply proxied body. A few informative ...` have me so confused
- we should probably have a header before the general error section
  - Something like:
      ```html
        <p>
        Endpoints with a non-optional <code>apiKey</code> parameter
        require a valid <code>apiKey</code> to work. It is recommended
        to pass the <code>apiKey</code> in the request body.
        </p>
        <h3>General Errors and Status Codes</h3>     
        <p>
	    In general, if a response has a negative <code>err</code> code
	    in a json body then there was an error issued by the api codebase. If
	    there is a positive <code>err</code> code in a json body, then
	    the error was issued by the application framework or web server and
	    the numeric value will match the http status code.
      </p>
     ```
      ?

- we might want headers above each endpoint as well
  - Done

Reply:
> > is this really a header?
> 
> I am assuming you're talking about the `<h1>` ? Yes

I meant the `<h4>` you have around the
> Depending on which domain the service is accessed through, there may or may not be a `/skitter` prefix to these endpoints. 

block. Is this semantically a section header, or should it be a `<p>`? It's more of a parenthetical note than a heading.


> As for the actual JSON, it's not semantic HTML but a random design choice.

What would you think of using `<blockquote><code>...</code></blockquote>`?

> > this doesn't pass a validator, but mine isn't free of typos either lol. We should fix it.
>
> I fixed the typos I could see

I meant html issues, of which parsing issues have been fixed (https://validator.w3.org/) , but there the use of headers and low contrast are still flagged on an accessibility tool (https://wave.webaim.org/).

> > we should probably have a header before the general error section
>
> Something like: 

yes

> > we might want headers above each endpoint as well
> 
> Done

thanks!

Reply:
-  should it be a `<p>`? 
   Yep. Changed it thanks.
- What would you think of using `<blockquote><code>...</code></blockquote>`?
  :eyes: well... I tried this and it looks weird although I tried this after changing the colours with the help of this [Contrast Checker]. 
  
  I attached two images in a `temp` folder located in  `/skitter_api/var/www/skitter.fanfic.dev/`.
    The [first image] shows  `<blockquote><code>...</code></blockquote>` while the [second image] shows the same image with one difference which is this css code block:
    ```css
    blockquote > code {
      background: #FFF;
    }
    ```
   
- ... use of headers and low contrast are still flagged on an accessibility tool
Fixed using [LightHouse] and [Contrast Checker]
-    >  we should probably have a header before the general error section
  
      Done

[LightHouse]: https://developers.google.com/web/tools/lighthouse#devtools
[Contrast Checker]: https://webaim.org/resources/contrastchecker/
[first image]: https://github.com/Jemeni11/skitter_api/blob/master/var/www/skitter.fanfic.dev/temp/background.png
[second image]: https://github.com/Jemeni11/skitter_api/blob/master/var/www/skitter.fanfic.dev/temp/no-background.png

Reply:
The contrast checker suggested making the foreground **and** background darker? What about using the new darker foreground color, but the original lighter background color -- that would be even higher contrast, but will still stand out from the non-`code` blocks near it.

Reply:
Hi, I'm sorry I'm late.
> What about using the new darker foreground color, but the original lighter background color ...

done

Reply:
No worries, I know how time can get :)

I think you accidentally changed the wrong color since the contrast on the latest version is much lower. I was suggesting to use the new darker foreground color:
```
    color: #510b15;
```
along with the original lighter background color:
```
    background: #eee;
```

That has a contrast of 12.84:1 versus the 2.18:1 the current version has.

Reply:
Doneee

Reply:
Thanks!